.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Closes # .
```

Comment:
I don't think that's desirable. Not all PRs close something.

Reply:
resolved.